contrib/builder/deb/amd64/generate.sh

```diff
@@ -42,11 +42,19 @@ for version in "${versions[@]}"; do
 	echo >> "$version/Dockerfile"
 
 	if [ "$distro" = "debian" ]; then
-		cat >> "$version/Dockerfile" <<-'EOF'
-			# allow replacing httpredir mirror
-			ARG APT_MIRROR=httpredir.debian.org
-			RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list
-		EOF
+		if [ "$suite" = "stretch" ]; then
+			cat >> "$version/Dockerfile" <<-'EOF'
+				# allow replacing deb mirror
+				ARG APT_MIRROR=deb.debian.org
+				RUN sed -i s/deb.debian.org/$APT_MIRROR/g /etc/apt/sources.list
```

Comment:
Thanks! I didn't check if my example worked :smile: @tianon's example (assuming that's the right syntax for sed), definitely +1